workflow/rules/bwa_mem.smk

```diff
@@ -8,7 +8,7 @@ __license__ = "GPL-3"
 
 rule bwa_mem:
     input:
-        reads=["pre-alignment/merge/{sample}_{unit}_R1.fq.gz", "Pre-alignment/merge/{sample}_{unit}_R2.fq.gz"],
+        reads=["pre-alignment/merge/{sample}_{unit}_R1.fq.gz", "pre-alignment/merge/{sample}_{unit}_R2.fq.gz"],
```

Comment:
I assume we leave the name `pre-alignment` then? Or could we change it to `prealignment`?

Reply:
The pre-alignment module will use "prealignment".

Reply:
👍

Reply:
changed in new version